Source/WebKit/UIProcess/Cocoa/XPCConnectionTerminationWatchdog.mm

```diff
@@ -47,7 +52,12 @@
     
 void XPCConnectionTerminationWatchdog::watchdogTimerFired()
 {
+#if HAVE(PROC_SIGNAL_WITH_AUDITTOKEN) && ENABLE(CLOSE_WEBCONTENT_XPC_CONNECTION_POST_LAUNCH)
+    auto returnValue = proc_signal_with_audittoken(&m_auditToken, SIGKILL);
+    RELEASE_LOG(Process, "XPCConnectionTerminationWatchdog::watchdogTimerFired(): proc_signal_with_audittoken returned %d", returnValue);
+#else
     terminateWithReason(m_xpcConnection.get(), ReasonCode::WatchdogTimerFired, "XPCConnectionTerminationWatchdog::watchdogTimerFired");
```

Comment:
It is very suspect for XPCConnectionTerminationWatchdog to be using another API to terminate processes than other code paths. Other code paths still rely on `terminateWithReason()`, which AFAICT will no longer work once `CLOSE_WEBCONTENT_XPC_CONNECTION_POST_LAUNCH` gets turned on.

Reply:
Ah, great catch, Chris! I will look closer at this.

Thanks for reviewing!